README.md

```diff
@@ -15,6 +15,8 @@ LoopBack 4 is a work in progress, the public API is frequently changed in
 backwards-incompatible ways. See [Upcoming-Releases on wiki](https://github.com/strongloop/loopback-next/wiki/Upcoming-Releases)
 for more details.
 
+_LB4 is recommended for use to learn the latest and greatest [features and technologies](https://loopback.io/doc/en/lb4/Crafting-LoopBack-4.html).  However, it is not recommended for production use yet. For production use, we recommend using [LoopBack 3](https://loopback.io/doc/en/lb3/index.html)._
```

Comment:
I wonder if it is better for rewording the first sentence as something like:
To learn about the latest and greatest LoopBack features and technologies for your next project, LoopBack 4 is recommended.

Reply:
We don't use terms such as `LB4` externally.